AppCenterDataStorage/AppCenterDataStorageTests/MSTokenTests.m

```diff
@@ -0,0 +1,28 @@
+
+#import <Foundation/Foundation.h>
+#import "MSTestFrameworks.h"
+
+
+static NSString *const partitionName = @"TestAppSecret";
+static NSString *const token = @"mockToken";
+
+@interface MSTokenTests : XCTestCase
+
+
+@end
+
+@implementation MSTokenTests
```

Comment:
You should fill unit tests to get this PR merged.

Reply:
We have a separate work item for these unit tests